### DISCUSSION TITLE
Kurviger Datei versenden

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#201] [Q&A] [answered]
Wenn ich eine kurviger Route importiere und per Mail versende wird diese in cruiser 1:1 wie gewünscht übernommen.
Wenn ich einen wegpunkt nur ganz wenig verschiebe wird die ganze Route neu berechnet und sieht dann komplett anders aus.
Lässt sich das irgendwie 
vermeiden ??

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

There are 2 different things:
- Navigation routing does not change the routes. It only calculates until next waypoint.
- Route planning currently calculates the whole route.
This will change in https://github.com/devemux86/cruiser/discussions/142

Das sind 2 verschiedene Dinge:
- Das Navigations-Routing ändert die Routen nicht. Es wird nur bis zum nächsten Wegpunkt berechnet.
- Die Routenplanung berechnet derzeit die gesamte Route.
Dies wird sich in https://github.com/devemux86/cruiser/discussions/142 ändern.

**Reply by 1maxi2** [reply]
Das heißt wenn ich einen Freund meine Route sende damit wir die gleiche Strecke fahren dann funktioniert das. Aber sobald er eine kleine Änderung macht um zum Beispiel zu einer Tankstelle zu fahren ist die ganze Route komplett verändert. Ist das so richtig und er darf keine ! noch so kleine Änderung vornehmen

**Reply by devemux86** [reply]
@1maxi2 

This is how all routers work: they re-calculate the whole route.
Unless the apps do their magic with smart route calculation.

In navigation the route does not change and you do not need to do anything.
Turn off rerouting, see the fuel stations on map, visit a station and return on the route.

---

So funktionieren alle Router: Sie berechnen die gesamte Route neu.
Es sei denn, die Apps zaubern mit intelligenter Routenberechnung.

Bei der Navigation ändert sich die Route nicht und Sie müssen nichts tun.
Schalten Sie das Rerouting aus, sehen Sie die Tankstellen auf der Karte, besuchen Sie eine Tankstelle und kehren Sie auf die Route zurück.

**Reply by 1maxi2** [reply]
https://kurv.gr/87QPS
Eigenartigerweise verändert die " alte" kurviger app die Route kaum.
Nur am Anfang routet er nicht mehr über die Autobahn obwohl zugelassen. Nach der Änderung bleibt die original Route fast unverändert.

**Reply by devemux86** [reply]
@1maxi2 

The kurv.gr links require online route calculation. Only the .kurviger files are offline route formats.

Die kurv.gr-Links erfordern eine Online-Routenberechnung. Nur die .kurviger-Dateien sind Offline-Routenformate.

**Comment by devemux86** [comment]
You can use the following BRouter (offline) settings and speed option to avoid main roads:

Sie können die folgenden BRouter-Einstellungen (offline) und die Geschwindigkeitsoption verwenden, um Hauptstraßen zu vermeiden:

<img src="https://github.com/devemux86/cruiser/assets/3484020/153ae992-b337-46a4-8a43-6a3993f0799a" width="50%"/>

**Reply by 1maxi2** [reply]
O.k.
